model/word2vec/state.go

```diff
@@ -223,9 +223,9 @@ func (s *State) trainRemainderBatch(wordIDs []int, trainOne func(wordIDs []int,
 func (s *State) incrementDoneWord() {
 	for range s.cwsCh {
 		s.trainedWords++
-		if s.trainedWords%s.BatchSize == 0 {
+		if s.trainedWords%s.batchSize == 0 {
 			s.lrMutex.Lock()
-			s.currentLearningRate = computeLearningRate(s.InitLearningRate, s.Theta, s.trainedWords, s.TotalFreq())
+			s.currentLearningRate = s.computeLearningRate(s.trainedWords, s.TotalFreq())
```

Comment:
if `computeLearningRate` is a method of `*State` then there is no real need to pass in `s.trainedWords` and `s.TotalFreq()`. The method can just grab the field of the `*State`.

And in that case, you should just call it `updateLearnRate`

Reply:
Sure to fix!